core/cloudflow-akka/src/main/resources/cassandrapersistence.conf

```diff
@@ -0,0 +1,45 @@
+# Adopted from https://github.com/akka/akka-persistence-cassandra/blob/master/example/src/main/resources/common.conf
+# maybe https://doc.akka.io/docs/akka-persistence-cassandra/current/configuration.html
+akka {
+  persistence {
+    journal {
+      plugin = "akka.persistence.cassandra.journal"
+      # to create the schema
+      auto-start-journals = ["akka.persistence.cassandra.journal"]
+    }
+    snapshot-store {
+      plugin = "akka.persistence.cassandra.snapshot"
+    }
+
+    cassandra {
+      journal {
+        class = "akka.persistence.cassandra.journal.CassandraJournal"
+        keyspace-autocreate = true
+        tables-autocreate = true
+      }
+
+      snapshot {
+        class = "akka.persistence.cassandra.snapshot.CassandraSnapshotStore"
+        keyspace-autocreate = true
+        tables-autocreate = true
+      }
+
+      query {
+        class = "akka.persistence.cassandra.query.CassandraReadJournalProvider"
+        refresh-interval = 2s
+      }
+    }
+  }
+}
+
+datastax-java-driver {
+  basic{
+    load-balancing-policy.local-datacenter = "datacenter1"
```

Comment:
Can this be hardcoded like this?

Reply:
Yes, it is recommended setting. Without setting this the code fails.